<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="E5470-01381" ignored="0" name="TestB1" tests="3" failures="1" timestamp="19 Apr 2018 22:25:36 GMT" time="0.043" errors="0">
  <testcase name="TestB11" time="0.009" classname="suiteB.TestB1"/>
  <testcase name="TestB12" time="0.004" classname="suiteB.TestB1">
    <failure type="java.lang.AssertionError" message="expected [B] but found [A]">
      <![CDATA[java.lang.AssertionError: expected [B] but found [A]
at suiteB.TestB1.TestB12(TestB1.java:19)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- TestB12 -->
  <testcase name="TestB13" time="0.0" classname="suiteB.TestB1">
    <skipped/>
  </testcase> <!-- TestB13 -->
</testsuite> <!-- TestB1 -->
